templates/compose/gotify.yaml

```
@@ -0,0 +1,31 @@
+# documentation: https://gotify.net/docs/install
+# slogan: Gotify is an open-source self-hosted notification server.
+# category: productivity
+# tags: productivity,notification,collaboration
+# logo: svgs/gotify.png
+# port: 80
+
+services:
+  gotify:
+    image: gotify/server
```

Please pin the version to a static one so future updates to the service won't break the template (user can manually update the version if they want)
```suggestion
    image: gotify/server:2.7.3
```